R/calcFeDemandIndustry.R

```diff
@@ -1285,7 +1287,64 @@ calcFeDemandIndustry <- function(scenarios, use_ODYM_RECC = FALSE, last_empirica
     ungroup() %>%
     mutate(value = .data$value * .data$share) %>%
     select("scenario", region = "iso3c", "year", item = "pf", "value") %>%
-    verify(expr = is.finite(.data$value), description = "Finite industry_subsectors_en values") %>%
+    verify(expr = is.finite(.data$value),
+           description = "Finite industry_subsectors_en values")
+
+  ### bespoke CHA electricity 2025 ----
```

Comment:
maybe add an additional sentence: 
"increase 2025 China industry electricity demand so that it matches extrapolation of 2018-2022 IEA data. This increase is then kept constant for all following time steps"